### PR TITLE
add azure cloud creds

### PR DESCRIPTION
**Pull request by yonasberhe23**
Adding variables for Azure credentials to be utilized in UI e2e test runs in Jenkins pipeline (existing and recurring runs)

### CODE REVIEW

File: templates/dashboard-tests/overlay/tmp/run.sh [hunk 1]
```diff
@@ -166,6 +166,9 @@ if [ ${CORRAL_rancher_type} = "existing" ]; then
       -e TEST_SKIP=setup \
       -e AWS_ACCESS_KEY_ID=${CORRAL_aws_access_key} \
       -e AWS_SECRET_ACCESS_KEY=${CORRAL_aws_secret_key} \
+      -e AZURE_CLIENT_ID=${CORRAL_azure_client_id} \
+      -e AZURE_CLIENT_SECRET=${CORRAL_azure_client_secret} \
+      -e AZURE_AKS_SUBSCRIPTION_ID=${CORRAL_azure_subscription_id} \
```
Comment (caliskanugur) [inline templates/dashboard-tests/overlay/tmp/run.sh:171]: Are these inputs already declared somewhere, in the template manifest or the package itself? or they don't even need that?

**Comment by izaac** [comment]
@caliskanugur  They are coming from Jenkins' then stored in the corral config hence the `CORRAL_` prefix and then they are used here feeding the docker container on a remote node.

**Comment by caliskanugur** [comment]
I see, it makes sense, thanks @izaac !

**Review by caliskanugur — APPROVED** [review]
Looks good to me

**Pull request merged**